recipe/meta.yaml

```diff
@@ -29,21 +29,21 @@ requirements:
   host:
     - python
     - pip
-    - conda-package-handling >=1.1
+    - conda-package-handling >=1.3
```

Comment:
I think we might need to add ``six`` in this section as well.
Because of ``setup.py`` is importing it.
https://github.com/conda/conda-build/blob/d45f7c77131333f5b48b4c97e93a3b82e62c9b66/setup.py#L2